public/app/features/browse-dashboards/components/BrowseActions/BrowseActions.tsx

```diff
@@ -130,10 +131,11 @@ export function BrowseActions() {
   return (
     <div className={styles.row} data-testid="manage-actions">
       <Button onClick={showMoveModal} variant="secondary">
-        Move
+        <Trans i18nKey="browse-action.move-button">Move</Trans>
```

Comment:
We should have 3 separate sections for these keys ([here](https://github.com/grafana/grafana/blob/main/contribute/internationalization.md#phrase-id-naming-convention) is the guide for it). So I would recommend changing these to `browse-dashboards.action.move-button` / `browse-dashboards.action.delete-button`, what do you think?